cpanfile

```diff
@@ -52,10 +52,12 @@ on 'test' => sub {
   requires 'Perl::Critic';
   requires 'Test::Output';
   requires 'Test::Fatal';
+  requires 'Test::Warnings';
   requires 'Test::Pod';
   requires 'Test::MockModule';
   requires 'Pod::Coverage';
   requires 'Devel::Cover';
+  requires 'String::Random';
```

Comment:
Ok, I didn't bother to check and this one we don't have even in TW, os-autoinst build is now failing because of that.

`t/10-test-image-conversion-benchmark.t` requires that.